package.json

```diff
@@ -14,10 +14,7 @@
   "bugs": {
     "url": "https://github.com/jonschlinkert/markdown-toc/issues"
   },
-  "license": {
-    "type": "MIT",
-    "url": "https://github.com/jonschlinkert/markdown-toc/blob/master/LICENSE"
-  },
+  "license": "",
```

Comment:
ah sorry, I just ran `npm init` to have npm fill in missing license details and I guess it didn't handle this. will fix